03/Vector3.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include "Vector3.h"
+
+
+Vector3::Vector3(int x, int y)
+	: mX(x)
+	, mY(y)
+{
+}
+
+int Vector3::Multiply(const Vector3& reference) const
+{
+	return (reference.mX * reference.mY) + (mX * mY);
```

Comment:
i want memberwise multiplication.. not dot product(and this is actually not even a dot product)